src/DeploymentManager/DeploymentManager/Commands/GetService.cs

```diff
@@ -61,20 +51,19 @@ public class GetService : DeploymentManagerBaseCmdlet
 
         [Parameter(
             Position = 2,
-            Mandatory = true, 
+            Mandatory = false, 
             ParameterSetName = DeploymentManagerBaseCmdlet.InteractiveParamSetName,
             HelpMessage = "The name of the service.")]
         [Parameter(
-            Position = 2,
-            Mandatory = true, 
+            Position = 1,
```

Comment:
Yes, the current positioning and the parameter sets needs to be fixed and it was a miss from our side the last time we published. 
We understand this is a breaking change and want to fix this. Please advise on how/when this can be released.

Reply:
> Yes, the current positioning and the parameter sets needs to be fixed and it was a miss from our side the last time we published.
> We understand this is a breaking change and want to fix this. Please advise on how/when this can be released.

Refer to another comment. I have listed the steps to publish a breaking change.

Reply:
Reverting all changes that touches parameter sets for all cmdlets other than Step (which is the new functionality).